src/sentry/web/frontend/react_page.py

```diff
@@ -38,6 +38,15 @@ def handle_react(self, request):
 # TODO(dcramer): once we implement basic auth hooks in React we can make this
 # generic
 class ReactPageView(OrganizationView, ReactMixin):
+    def handle_auth_required(self, request, *args, **kwargs):
+        # If user is a superuser (but not active, because otherwise this method would never be called)
+        # Then allow client to handle the route and respond to any API request errors
+        if request.user.is_superuser:
```

Comment:
was going to say that i think it's possible for request.user to be anonymous here, but apparently is_superuser will always be false in that case https://docs.djangoproject.com/en/1.7/ref/contrib/auth/#anonymous-users so this should be fine